js/src/functions.js

```diff
@@ -171,7 +171,7 @@ Functions.addDatepicker = function ($thisElement, type, options) {
                 if (typeof tooltip !== 'undefined') {
                     tooltip.disable();
                     var $note = $('<p class="note"></div>');
-                    $note.text(tooltip.option('content'));
+                    $note.text('Note: ' + tooltip.option('content'));
```

Comment:
This is not correct, phpMyAdmin is a translated tool you need to use: `libraries/classes/Controllers/JavaScriptMessagesController.php`
It would look like that
```suggestion
                    $note.text(Messages.strNote + tooltip.option('content'));
```

Reply:
Missed it, I was looking at the libraries/classes/Controllers/JavaScriptMessagesController.php and was thinking about it. Will do that change needed. Thanks for pointing out.